### DISCUSSION TITLE
Question about AnyLogic versions

### DISCUSSION THREAD

**Discussion by xylhal** [the-anylogic-company/Alpyne#29] [Q&A] [answered]
Hi,

I first started out with AnyLogic PLE version to experiment but I now have access to AnyLogic Professional version installed as well.

How do I know which version of AnyLogic is running when running the Alpyne client? Is the version dependent on which version on AnyLogic the RLExperiment is exported from?

**Comment by t-wolfeadam** [comment, marked as answer]
AFAIK, there's unfortunately there is no way to detect this; exported models don't have any indicator (public facing or even on my end) of what edition is was exported from. 

Though, the execution of the model is the same whether it's from PLE or Professional, so this shouldn't be an issue. Alpyne also uses your exported model for the vast majority of libraries, so versioning shouldn't be an issue either.
